gen.py

```diff
@@ -128,29 +151,39 @@ def write_player(path, bits):
   <img src="http://mlb.mlb.com/mlb/images/players/head_shot/{id}.jpg" alt="{name}">
 </p>""".format(
         name=name,
-        id=data.pop('id')
+        id=info.get('id')
     ).strip()
 
     cols, sections = set(), {'misc':{}}
-    for key, value in data.iteritems():
-        if not isinstance(value, dict):
-            continue
-
-        year = unicode(key[0:4])
-        if len(key) > 3 and year.isnumeric():
```

Comment:
This is where the list of years was built before.  May be worth adopting again.  But it looks like the format of the months has changed.  You may be able to depend on the ordering of the CSV here.